CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_fivetran_log v1.3.0
+
+## Breaking Changes
+- Deprecated the `transformation` and `trigger_table` source tables and any downstream transforms. These tables only housed information on Fivetran Basic SQL Transformations, which were sunset last year ([PR #96](https://github.com/fivetran/dbt_fivetran_log/pull/96)).
```

Comment:
We should probably mention the end model in particular that is being removed as well.

Additionally, let's call out the variable being removed for now as well.